.travis.yml

```diff
@@ -20,6 +21,9 @@ matrix:
   exclude:
     - env: JOB=smoke
       node_js: "0.12"
```

Comment:
It looks like this is no longer excluding smoke jobs with nodejs 0.12 :(

That's not the end of the world - can we try it with:

```
exclude:
  - env: JOB-smoke
    node_js: "0.12"
  - env: JOB-smoke
    node_js "0.10"